### ISSUE TITLE
Clearer instructions for Shadertoy.com import

### ISSUE THREAD

**Issue by y4my4my4m** [y4my4my4m/kde-shader-wallpaper#16]
Should instruct how to import, give user feedback for loading/errors.

**Comment by TheCorruptCat** [comment]
how do you import them from shadertoy.com?

**Comment by y4my4my4m** [comment]
@TheCorruptCat you simply copy the URL into the shader path or just the "id" of the shadertoy
**Note: this doesnt work for shaders with buffers!**

**Comment by y4my4my4m** [comment]
@thembees this is only for Plasma5 version, but I do mean in the actual configuration GUI
![image](https://github.com/y4my4my4m/kde-shader-wallpaper/assets/8145020/ecb0bb74-f56e-41d5-91a0-a86cdbb044fb)

You simply put the URL of the shadertoy.com you want to use, or it's ID.

Be aware that **this only works for shaders not using buffers!**

Test it with this one for example: https://www.shadertoy.com/view/4td3zj
Copy paste that into the path and then press enter and/or apply.

**Comment by thembees** [comment]
Oh thank you, i'm on plasma6 so that's why I didn't see it!

**Comment by y4my4my4m** [comment]
> Oh thank you, i'm on plasma6 so that's why I didn't see it!

yeah, plasma6 doesn't support shadertoy import as of yet as shaders need to be compiled.
I'm planning to have an automatic convert/processor/compiler tool though.

**Comment by Steinardarri** [comment]
I am trying to manually compile a shader from shadertoy but I am having trouble with the global variables. How do you handle them for Spirv?

**Comment by OpenJowel** [comment]
Hi,

Sorry for bumping here. I've also been trying to compile unsuccessfully with qsb but even when i take some  ```.frag``` that come with the module. So I think I'm somehow missing an information.

Where can i find the "python scripts" above-mentioned ? It looks helpful

I'm also on plasma6. Besides that, everything works great with the built-in shader. I love this KDE module !
I just want to load/compile more shaders but currently don't know how.

Thank you very much if anyone can help

**Comment by y4my4my4m** [comment]
@OpenJowel glad you solved it.

Ive been meaning to make plasma6 the default branch as most people have switched it seems.

I should probably revise and include the python scripts in the package itself too.
